src/sentry/static/sentry/app/utils/discover/charts.tsx

```diff
@@ -0,0 +1,39 @@
+import {getDuration, formatPercentage} from 'app/utils/formatters';
+import {aggregateOutputType} from 'app/utils/discover/fields';
+
+/**
+ * Formatter for chart tooltips that handle a variety of discover result values
+ */
+export function tooltipFormatter(value: number, seriesName: string): string {
+  switch (aggregateOutputType(seriesName)) {
+    case 'number':
```

Comment:
Did you miss a `case 'integer':` here or was that intended?

Reply:
It was missing from the code this was drawn from. We don't have any aggregates that create integer types right now, but it is good to have for completeness.